Dockerfile

```diff
@@ -14,6 +14,8 @@ WORKDIR /usr/src/multiqc
 # Install MultiQC
 RUN python -m pip install .
 
+ENV PATH "$PATH:/usr/src/multiqc"
```

Comment:
Should this not be `/usr/src/` if it's the directory that contains the `multiqc`binary? This looks like the full path to the binary to me..?